test/voteInOrganizationScheme.js

```diff
@@ -0,0 +1,109 @@
+import { DAO } from "../lib/dao";
+import { VoteInOrganizationScheme } from "../lib/contracts/voteInOrganizationScheme";
+import { getVotingMachineForScheme } from "./helpers";
+import { Utils } from "../lib/utils";
+
+const createProposal = async function () {
+
+  const originalDao = await DAO.new({
+    name: "Original",
+    tokenName: "Tokens of Original",
+    tokenSymbol: "ORG",
+    schemes: [
+      { name: "ContributionReward" }
+      , { name: "SchemeRegistrar" }
+    ],
+    founders: [{
+      address: accounts[0],
+      reputation: web3.toWei(1000),
+      tokens: web3.toWei(1000)
+    }]
+  });
+
+  const schemeToDelete = (await originalDao.getSchemes("ContributionReward"))[0].address;
+  assert.isOk(schemeToDelete);
+  const schemeRegistrar = await originalDao.getScheme("SchemeRegistrar");
+  assert.isOk(schemeRegistrar);
+  const result = await schemeRegistrar.proposeToRemoveScheme(
+    {
+      avatar: originalDao.avatar.address,
+      scheme: schemeToDelete
+    });
+
+  assert.isOk(result.proposalId);
+
+  /**
+       * get the voting machine will be used for this proposal
+       */
+  const votingMachine = await getVotingMachineForScheme(originalDao, schemeRegistrar, 2);
+
+  assert.isOk(votingMachine);
+
+  return { proposalId: result.proposalId, votingMachine: votingMachine.address };
+};
+
+describe("VoteInOrganizationScheme", () => {
+  let dao;
+  let voteInOrganizationScheme;
+  beforeEach(async () => {
+
+    dao = await DAO.new({
```

Comment:
This code(DAO.new) is duplicate at vestingScheme (and probably on other places) .
Should it be moved to a common place (helpers.js ).

Reply:
There is potential in that, but it is not the subject of this PR.

Reply:
I am having too many issues with PRs that balloon when I have other PRs waiting behind that have to be merged, as well as PRs taking a long time to get approved.  Refactoring test code can easily and be done better in isolation.  This PR is about adding a wrapper.  How about we do a separate one on cleaning up test code (an ongoing process anyways...)?